packages/client-api-schema/src/types.ts

```diff
@@ -1,3 +1,9 @@
+/**
+ * Ethereum Address
+ * @pattern  ^0x([a-fA-F0-9]{40})|0$
+ */
+type Address = string;
```

Comment:
I think if you do `export type Address = string` it will generate a definition in the schema for it.